### DISCUSSION TITLE
首页标语（Ikun语录）推荐

### DISCUSSION THREAD

**Discussion by ESCATS** [ikun-svelte/ikun-ui#198] [Ideas]
刚进入官网立马弹出"哇！真的是你呀"
关闭官网时弹出"你干嘛 哈哈 哎呦"
长时间观看ikun-ui文档弹出"再多看一眼就爆炸！"

欢迎大家还有什么好的语录及官网优化推荐

**Comment by ESCATS** [comment]
可以顶置吗😀😀

**Comment by baiwusanyu-c** [comment]
有点意思，来个pr吗

**Comment by moshuchao** [comment]
🐂🍺

**Comment by fuchanghai** [comment]
在右侧加串 字，您的练习时长为：xxx。随动

**Comment by hb2cpc** [comment]
📞蔡徐坤向您发起了篮球邀请

⚪接受 ⚪ 拒绝 

🔘和篮球一起打蔡徐坤
